zstacklib/zstacklib/utils/qemu.py

```
@@ -120,3 +120,35 @@ def _parse_version(version_output):
+00000030:  31 31 31 31 31 0a 31 31 31 31 31 31 31 31 0a 31  11111.11111111.1
+00000040:  31 31 31 31 31 31 31 0a 31 31 31 31 31 31 31 31  1111111.11111111
+00000050:  0a 31 31 31 31 31 31 31 31 0a 31 31 31 31 31 31  .11111111.111111
+00000060:  31 31 0a 31 11 11 11 11 11 11 11 11 11 11 11 11  11.1............
+00000070:  11 11 11 11 11 11 11 11 11 11 11 11 11 11 11 11  ................
+read 128/128 bytes at offset 0
+128 bytes, 1 ops; 00.04 sec (3.405 KiB/sec and 27.2394 ops/sec)
+
+"""
+
+def read_image_content(image_path, offset, length, format):
+    l = (length + 15) & ~15  # align to 16 bytes
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)